server/trips/trips.js

```diff
@@ -20,4 +23,16 @@ UserTrip.sync();
 Trip.sync();
 User.sync();
 
+// Model functions
+
+Trip.createTrip = (name, user, friends) =>
+  Trip.create({ name })
+  .then(trip => {
+    trip.addUsers([...friends, user]);
```

Comment:
change this block to `return trip.addUsers...` and remove below return statement